sonar-project.properties

```diff
@@ -0,0 +1,3 @@
+sonar.projectKey=pagopa_developer-portal
+sonar.organization=pagopa
+sonar.sources=apps/**/src, packages/gitbook-docs/src
```

Comment:
Why not as following?

```suggestion
sonar.sources=apps/**/src, packages/**/src
```

Reply:
I wasn't sure that running the scan on `eslint-config` package would give useful information.